yammbs/_minimize.py

```diff
@@ -61,7 +61,7 @@ def _minimize_blob(
     force_field: str,
     n_processes: int = 2,
     chunksize=32,
-) -> dict[str, list["MinimizationResult"]]:
+) -> Iterator["MinimizationResult"]:
```

Comment:
This was probably simply wrong before?

Reply:
Yeah I thought so too. I think this function used to return a dict before it switched to yielding.

Reply:
That tracks with my memory